src/sentry/web/frontend/organization_settings.py

```diff
@@ -57,17 +90,32 @@ def get_form(self, request, organization):
                 'allow_joinleave': bool(organization.flags.allow_joinleave),
                 'enhanced_privacy': bool(organization.flags.enhanced_privacy),
                 'allow_shared_issues': bool(not organization.flags.disable_shared_issues),
+                'scrub_data': bool(organization.get_option('sentry:scrub_data', False)),
```

Comment:
They're not.

I load the settings page, it has the box for "scrub data" checked or not checked. Either way it's wrong. Scrub data is "Data scrubbing is enabled for all projects" or "Each project can determine what they want to do". If I untick the box, what do I assume? That it's disabled org wide? That its simply not-enabled? Realistically this is a three-tier setting, but we don't want one of those awful three-tier checkboxes, and we don't want the "disabled for everyone" setting.

Right now the code is treating it as inherited options -- which given how it deletes the option if its not set, works. If I were to however set it to False vs True though, that'd obviously not work, and because they're named identically its confusing. We should simply adjust the names/patterns based on that, because otherwise this code will be too confusing for anyone else to maintain (and its data-based so we're stuck with it).